android/src/main/java/com/visioncameraocr/VisionCameraOcrPackage.kt

```diff
@@ -1,19 +1,22 @@
-package com.visioncameraocr
+ package com.visioncameraocr
 
-import com.facebook.react.ReactPackage
-import com.facebook.react.bridge.NativeModule
-import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.uimanager.ViewManager
-import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+ import com.facebook.react.ReactPackage
+ import com.facebook.react.bridge.NativeModule
+ import com.facebook.react.bridge.ReactApplicationContext
+ import com.facebook.react.uimanager.ViewManager
 
+ import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+ import com.mrousavy.camera.frameprocessor.FrameProcessorPluginRegistry;
 
-class VisionCameraOcrPackage : ReactPackage {
-    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
-        FrameProcessorPlugin.register(OCRFrameProcessorPlugin())
-        return emptyList()
-    }
+ class VisionCameraOcrPackage : ReactPackage {
```

Comment:
I'm extremely green with Kotlin, but I believe you'd need to do something like this:

```
class VisionCameraOcrPackage : ReactPackage {

    companion object {
        init {
            FrameProcessorPluginRegistry.addFrameProcessorPlugin("scanOCR") {
                OCRFrameProcessorPlugin(it)
            }
        }
    }

    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
        return emptyList()
    }
    //...
}
```

in order to avoid the "Tried to add a Frame Processor Plugin with a name that already exists!" error that @xulihang (and myself) are facing

Reply:
Sorry for the late response, I've been away from the code for a bit. 

I have it in my personal app (I started the ocr bit from the example app), it happens on Android whenever I reload my development build. 

It happens even if I don't import this package in my code, which leads me to believe it's an issue with the native file for the build. I also don't get this error on iOS.